frontend/src/assets/kir-dev-logo.tsx

```diff
@@ -0,0 +1,209 @@
+/*eslint-disable max-len*/
+import type { CSSProperties } from 'react'
+
+// this file is not .svg because babel had issues parsing the style attributes with custom properties.
+// (.svg files that are imported as modules get transformed into jsx under the hood)
+// another thing: if you want to display the animated version of this, you have to place the svg in the dom as a <svg> instead of an <img> tag
+// that's why we need this file
+export const KirDevLogo = () => (
```

Comment:
Ide lehetne egy props: SVGProps vagy hasonlót és belecsorgatni {...props} kifejezéssel az SVG-be, így kívülről nagyobb ráhatásod tud lenni.

Reply:
Done